AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Subscribe/DataStorePublisher.swift

```diff
@@ -8,18 +8,21 @@
 import Amplify
 import Combine
 
-// TODO: Should this be a multicast publisher?
 @available(iOS 13.0, *)
-struct DataStorePublisher: DataStoreSubscribeBehavior {
+struct DataStorePublisher: ModelSubcriptionBehavior {
 
     private let subject = PassthroughSubject<MutationEvent, DataStoreError>()
 
-    func publisher<M: Model>(for modelType: M.Type) -> AnyPublisher<MutationEvent, DataStoreError> {
+    func publisher(for modelName: ModelName) -> AnyPublisher<MutationEvent, DataStoreError> {
```

Comment:
As noted above, let's just remove this API and let call sites filter according to their use cases

Reply:
Removed